pyro/ops/tensor_utils.py

```diff
@@ -393,15 +394,15 @@ def inverse_haar_transform(x):
     return x
 
 
-def cholesky(x):
+def safe_cholesky(x):
     if x.size(-1) == 1:
         x = x.clamp(min=torch.finfo(x.dtype).tiny)
         return x.sqrt()
 
     # Add adaptive jitter.
     x = x.clone()
     x_max = x.data.reshape(*x.shape[:-2], -1).abs().max(-1, True).values
-    jitter = x_max * torch.finfo(x.dtype).eps
+    jitter = CHOLESKY_JITTER * torch.finfo(x.dtype).eps * x_max
```

Comment:
Adding jitter relatives to a specific "norm" (or something else) of the matrix makes sense to me. Probably renaming `CHOLESKY_JITTER` into `RELATIVE_CHOLESKY_JITTER`?